### DISCUSSION TITLE
What about Festivals?

### DISCUSSION THREAD

**Discussion by MarkJaroski** [jakopako/croncert-config#155] [Q&A] [answered]
Example: https://cullyjazz.ch/fr/programme

Is this kind of contribution welcome? Most of the venues only exist during the festival, but they are the same from one year to another, including theme: eg. Dr Schlagzeug is always for Swiss-german bands, etc. etc.

**Comment by jakopako** [comment, marked as answer]
I've not really thought about festivals yet, but yeah, I think it's a good idea! Feel free to add festivals too :)
